README.md

```diff
@@ -132,13 +132,14 @@ Some differences between inih and Python's [ConfigParser](http://docs.python.org
 ## Meson notes ##
 
 * The `meson.build` file is not required to use or compile inih, its main purpose is for distributions.
-* By default Meson only creates a static library for inih, but Meson can be used to configure this behavior:
-* with `-Ddefault_library=shared` a shared library is build.
-* with `-Ddistro_install=true` the library will be installed with the header and a pkg-config entry, you may want to set `-Ddefault_library=shared` when using this.
-* with `-Dwith_INIReader` you can build (and install if selected) the C++ library.
-* all compile-time options are implemented in Meson as well, you can take a look at [meson_options.txt](https://github.com/benhoyt/inih/blob/master/meson_options.txt) for their definition. These won't work if `distro_install` is set to `true`.
+* By default Meson is set up for distro installation, but this behavior can be configured for embedded use cases:
+* with `-Ddefault_library=static` static libraries are built.
```

Comment:
Thanks for this. One little tweak: can you please indent the three "with" bullets -- they should be sub-points of the "By default" bullet, right?

Reply:
Done 👍🏼